src/Microsoft.VisualStudio.ProjectSystem.CSharp.UnitTests/ProjectSystem/PropertiesFolderProjectTreePropertiesProviderTests.cs

```diff
@@ -216,24 +216,42 @@ public void ChangePropertyValues_TreeWithNestedPropertiesFolder_ReturnsUnmodifie
         [InlineData(@"
 Root(flags: {ProjectRoot})
     Properties (flags: {Folder AppDesignerFolder BubbleUp})
+", @"
+Root(flags: {ProjectRoot})
+    Properties (flags: {Folder AppDesignerFolder BubbleUp})
 ")]
         [InlineData(@"
 Root(flags: {ProjectRoot})
     Properties (flags: {Folder AppDesignerFolder})
+", @"
+Root(flags: {ProjectRoot})
+    Properties (flags: {Folder AppDesignerFolder BubbleUp})
+")]
+        [InlineData(@"
+Root(flags: {ProjectRoot})
+    Properties (flags: {Folder BubbleUp})
+", @"
+Root(flags: {ProjectRoot})
+    Properties (flags: {Folder AppDesignerFolder BubbleUp})
 ")]
         [InlineData(@"
 Root(flags: {ProjectRoot})
     Properties (flags: {Folder Unrecognized AppDesignerFolder})
+", @"
+Root(flags: {ProjectRoot})
+    Properties (flags: {Folder Unrecognized AppDesignerFolder BubbleUp})
 ")]
-        public void ChangePropertyValues_TreeWithPropertiesCandidateAlreadyMarkedAsAppDesigner_ReturnsUnmodifiedTree(string input)
+        public void ChangePropertyValues_TreeWithPropertiesCandidateAlreadyMarkedAsAppDesignerOrBubbleup_AddsRemainingFlags(string input, string expected)
```

Comment:
This tests and the My Project equivalent were left over from the previous implementation of this properties provider via (IProjectTreeModifier)- in that world, you got passed in existing trees, so you wanted to avoid modifying them for perf reasons - in the new model we only get called for new trees - so we always want to tag the AppDesigner with appropriate capabilities even if someone already did it.